ckanext/cfpb_extrafields/plugin.py

```diff
@@ -80,12 +83,14 @@ def _delete_and_rebuild_datadict(self, resource):
                 ds.delete_datastore_json(resource['id'], 'datadict')
             # don't fail if the filter is bad! (e.g., title_colname doesn't exist)
             except (tk.ObjectNotFound, tk.ValidationError), err:
+                # code review: write tests for this.
```

Comment:
In the SAG review Lamin suggested logging in case this pass results in problems. There should also be unit tests for this ultimately.